docs/source/mayavi/conf.py

```diff
@@ -15,7 +15,10 @@
 # Adding the current directory to the path, so that sphinx finds the
 # extensions.
 import os
-
+try:
+    basedir
```

Comment:
I am curious as to why this is needed?  Can't you just define it?  If there is a non-obvious reason, then please do write a comment explaining why this is needed.

Reply:
Thank you for your comment. When the building of document in mayavi-doc-translations.git, basedir is set by the follwing conf.py.
https://github.com/getfem-doc/mayavi-doc-translations/blob/master/conf.py
That's the reason to check the variable.